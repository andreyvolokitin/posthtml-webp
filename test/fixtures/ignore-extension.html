<!doctype html>
<html>
    <body>
        <img src="photo.svg">
        <img src="photo.gif">
        <img src="photo.jpg">
        <img src="photo.png">
    </body>
</html>
