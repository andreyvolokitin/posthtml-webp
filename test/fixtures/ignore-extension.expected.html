<!doctype html>
<html>
    <body>
        <img src="photo.svg">
        <img src="photo.gif">
        <picture><source type="image/webp" srcset="photo.jpg.webp"><img src="photo.jpg"></picture>
        <picture><source type="image/webp" srcset="photo.png.webp"><img src="photo.png"></picture>
    </body>
</html>
